docs/remote.md

```diff
@@ -151,6 +151,26 @@ JSON or XML are accepted. Other content-types are blocked the time to evaluate
 if they are useful and their security implication (javascript is probably not
 something we want to allow).
 
+### GET `/remote/_all_doctypes`
```

Comment:
Why the leading `_all_`? Does it carry special meaning?

Reply:
The stack has a `/data/_all_doctypes` endpoint (https://docs.cozy.io/en/cozy-stack/data-system/#list-the-known-doctypes), and I copied the `_all_` prefix from there.